app/dashboard/models.py

```diff
@@ -221,6 +221,7 @@ class Bounty(SuperModel):
     token_address = models.CharField(max_length=50)
     bounty_type = models.CharField(max_length=50, choices=BOUNTY_TYPES, blank=True)
     project_length = models.CharField(max_length=50, choices=PROJECT_LENGTHS, blank=True)
+    estimated_hours = models.IntegerField(blank=True, null=True)
```

Comment:
Should we use a `PositiveIntegerField` here?

Reply:
great idea! updated